docs/rfc/rfc-1.adoc

```diff
@@ -401,12 +412,140 @@ to submit multiple redemptions. After a redemption, if the wallet has under
 `wallet_min_btc` remaining, it transfers that BTC to a randomly selected wallet
 and closes.
 
+Each redemption request is identified by a concatenation of the wallet's pubkey
+hash and redeemer's output hash (redeemer's BTC address). Such an identifier
+allows retrieving pending redemption requests in a gas-efficient way based on
+information provided in the redemption proof. A consequence of this approach is
+that the redeemer can not use the same redemption BTC address when there is
+already one redemption request pending from the given wallet. User experience
+can be improved by the dApp by selecting the previous-oldest wallet without a
+pending redemption request to the given BTC address in case a new redemption is
+requested before the pending one gets cleared out.
+
+At a minimum, pending redemption request needs to capture the following
+information:
+- Expected range of value for redemption UTXO: needed to validate the amount
+  released by the system when processing the redemption proof. Fees are
+  governable and the expected amount redeemed needs to be captured at the moment
+  of the redemption request.
```

Comment:
> Expected range of value for redemption UTXO

I don't fully understand this, why are we talking about a range here? I'm not sure I get the intention of this paragraph right, but how I understand it is that we want to make sure that the share of the `Y` fee charged from the redeemer will be computed based on the value of `Y` at the moment of redemption request (so that if `Y` got increased by the governance before redemption request got processed, redeemer would not be charged more than he initially anticipated). But I don't see how that corresponds to need for storing an expected range for value of UTXO and how would that range be calculated.

Reply:
Say that I want to redeem 5 TBTC from my balance.

Right now, the redemption fee for the treasury is 0.1 TBTC.

I know that I  will receive no more than 4.9 BTC because 0.1 TBTC was a redemption fee paid to the system.

"No more" than 4.9 BTC because there is Bitoin fee paid to the miners. If that fee is 3 sat, then I will receive `4.9 BTC - 3 sat`.

At the moment of requesting the redemption, we do not know what the Bitcoin miner fee will be. We also do not know if the treasury fee is not going to be altered. That's why we either need to capture expected ranges or capture max Bitcoin fee and treasury fee and calculate if UTXO value (BTC I received) is above the acceptable value or not.